geps/gep-1897/index.md

```diff
@@ -5,61 +5,51 @@
 
 ## TLDR
 
-This document specifically addresses the topic of conveying HTTPS from the Gateway
-dataplane to the backend (backend TLS termination), and intends to satisfy the single
-use case “As a client implementation of Gateway API, I need to know how to connect to
-a backend pod that has its own certificate”. TLS configuration can be a nebulous topic,
-so in order to drive resolution this GEP focuses only on this single piece of functionality.
+This document aims to address the use case of origination of TLS to backends from implementations.
+This is accomplished by a new `BackendTLSPolicy` API.
```

Comment:
For clarity, please include the original use case plus your new use case(s).  Chihiro should be able to read this document and know what API components are affected.